### ISSUE TITLE
[CRASH] Crash of the game when dismantling the ion booster Space Exploration

### ISSUE THREAD

**Issue by Eldrinn-Elantey** [InappropriatePenguin/production-analyst#2]
```
24583.203 Error MainLoop.cpp:1284: Exception at tick 56756783: Мод Production Analyst (1.1.7) вызвал неустранимую ошибку.
Пожалуйста, сообщите об этой ошибке автору мода.

Error while running event production-analyst::on_player_mined_entity (ID 67)
__production-analyst__/control.lua:348: bad argument #2 of 2 to 'table_size' (table expected, got nil)
stack traceback:
	[C]: in function 'table_size'
	__production-analyst__/control.lua:348: in function <__production-analyst__/control.lua:337>
24583.203 Error ServerMultiplayerManager.cpp:92: MultiplayerManager failed: "Мод Production Analyst (1.1.7) вызвал неустранимую ошибку.
Пожалуйста, сообщите об этой ошибке автору мода.

Error while running event production-analyst::on_player_mined_entity (ID 67)
__production-analyst__/control.lua:348: bad argument #2 of 2 to 'table_size' (table expected, got nil)
stack traceback:
	[C]: in function 'table_size'
	__production-analyst__/control.lua:348: in function <__production-analyst__/control.lua:337>"
```

Mod list:
AAI_Language_Pack_1.1.12
aai-containers_0.2.11
aai-industry_0.5.19
aai-signal-transmission_0.4.7
Actual_Craft_Time_0.5.9
AdvancedArtilleryRemotesContinued_2.1.0
alien-biomes_0.6.7
assemblyanalyst_1.1.11
AutoDeconstruct_0.3.3
auto-deploy-robots-1dot1_0.0.5
Automatic_Train_Painter_1.1.4
automatic-logistic-chests_1.2.0
automatic-station-painter_1.0.5
AutoTrash_5.3.13
beautiful_bridge_railway_1.1.1
belt-balancer-performance_3.1.3
blueprint_reader_1.0.3
bobinserters_1.1.5
boblibrary_1.1.5
boblocale_1.1.5
BottleneckLite_1.2.3
bullet-trails_0.6.2
ChangeMapSettings_5.0.1
ColorCodedPlanners_1.0.1
combat-mechanics-overhaul_0.6.22
ConstructionPlanner_1.0.5
Constructron-Continued_1.0.32
CopyPasteModules_0.1.1
DragonIndustries_1.105.1
early_construction_0.9.3
Enhanced_Map_Colors_1.5.5
equipment-gantry_0.1.5
EquipmentGridLogisticModule_1.1.0
even-distribution_1.0.10
factorio-crash-site_1.0.2
factoripedia_0.0.9
factorissimo-2-notnotmelon_1.1.19
FactorySearch_1.9.2
Fill4Me-fixed_0.10.1
Filter_Combinator_1.1.7
flib_0.10.1
FluidMustFlow_1.3.1
FluidMustFlowSE_0.0.1
Geothermal_1.105.1
ghost-counter_1.1.9
GhostScanner_1.6.5
grappling-gun_0.3.2
GUI_Unifyer_1.12.0
helmod_0.12.11
Infinizoom_0.1.9
informatron_0.2.2
inserter-throughput_0.1.3
inventory-repair_19.1.0
jetpack_0.3.5
Krastorio2_1.3.4
Krastorio2Assets_1.2.0
Landfill_plus_1.0.19
LogisticTrainNetwork_1.16.10
LTN_Combinator_Modernized_1.0.11
LTN_Content_Reader_1.0.2
LTN_Language_Pack_1.9.5
ltn-cleanup_1.0.17
LtnManager_0.4.15
manual-inventory-sort_2.2.5
map-tag-generator_1.2.4
MaxRateCalculator_3.4.48
ModuleInserterER_5.2.5
MouseOverConstruction_1.2.0
multi-product-recipe-details_1.1.1
nco-AdvancedAirPurification_0.0.2
nco-InserterCranes_0.0.7
nixie-tubes_1.1.3
Noxys_Trees_0.4.2
ore-manager-fixed_1.1.1
Orphan Finder_1.1.3
OutpostPlanner1-1_1.2.3
PipeVisualizer_1.0.4
PlannerCore1-1_0.3.2
power-grid-comb_1.1.0
production-analyst_1.1.7
pump_1.1.3
qol_research_jatmn_3.3.3
QuickItemSearch_2.1.5
railloader_1.1.6
RateCalculator_2.4.4
Realistic_Heat_Glow_2.1.0
RealisticDecorationCleanup_1.1.1
RealisticOres_7.101.0
RealisticReactorGlow_1.101.7
RecipeBook_3.4.5
RepeatingIslands_Comp11_0.1.1
robot_attrition_0.5.12
Robot256Lib_1.1.4
rocket-log_1.1.9
rusty-locale_1.0.16
scootys-armor-swap_1.1.0
shield-projector_0.1.6
show-max-underground-distance_0.0.8
simhelper_1.1.4
solar-calc_0.3.11
some-autoresearch_1.1.0
sonaxaton-resource-map-markers_0.3.5
space-exploration_0.6.82
space-exploration-graphics_0.6.11
space-exploration-graphics-2_0.6.1
space-exploration-graphics-3_0.6.1
space-exploration-graphics-4_0.6.2
space-exploration-graphics-5_0.6.1
space-exploration-menu-simulations_0.6.8
space-exploration-postprocess_0.6.18
space-spidertron_0.2.3
stack-combinator_1.2.13
stack-size-tooltip_1.0.0
stdlib_1.4.7
TaskList_0.4.2
textplates_0.6.9
trainsaver_0.0.18
VehicleWagon2_3.1.21
wret-map-colors-fix_1.0.1
YARM_0.8.207

Log:

[factorio-previous.log](https://github.com/InappropriatePenguin/production-analyst/files/9284221/factorio-previous.log)

**Comment by Eldrinn-Elantey** [comment]
If you need anything else let me know

**Comment by InappropriatePenguin** [comment]
Thanks, this should now be fixed in 1.1.8.
